### PR TITLE
MCR-1396 IIIF-Presentation-API

### PR DESCRIPTION
**Pull request by sebhofmann**
Added first draft of IIIF-Presentation-API

### CODE REVIEW

**Review by rsteph-de — APPROVED** [review]
please set MyCoRe/GPL header comment before merging this component

File: mycore-iiif/src/main/java/org/mycore/iiif/image/resources/MCRIIIFImageResource.java [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+@Path("/iiif/image")
+public class MCRIIIFImageResource {
+    public static final String IIIF_IMAGE_API_2_LEVEL2 = "http://iiif.io/api/image/2/level2.json";
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    @GET
+    @Produces("application/ld+json")
+    @Path("{impl}/{identifier}/info.json")
```
Comment (yagee-de) [inline mycore-iiif/src/main/java/org/mycore/iiif/image/resources/MCRIIIFImageResource.java]: every method starts with `impl` path parameter. Should be path parameter of class

File: mycore-iiif/src/main/java/org/mycore/iiif/presentation/resources/MCRIIIFPresentationResource.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        @Produces(APPLICATION_LD_JSON)
+        @Path("{impl}/collection/{name}")
+        public Response getCollection(@PathParam(IMPL) String impl, @PathParam("name") String name) {
+            return null;
+        }
+    */
+
+    @GET
+    @Produces(APPLICATION_LD_JSON)
+    @Path("{impl}/{identifier}/manifest")
```
Comment (yagee-de) [inline mycore-iiif/src/main/java/org/mycore/iiif/presentation/resources/MCRIIIFPresentationResource.java]: every method starts with `impl` path parameter. Should be path parameter of class

**Comment by yagee-de** [comment]
:shipit: ?

**Comment by sebhofmann** [comment]
:thumbsup:

**Pull request merged**